ocs_ci/utility/rosa.py

```diff
@@ -46,10 +46,11 @@ def create_cluster(cluster_name, version):
     region = config.DEPLOYMENT["region"]
     compute_nodes = config.ENV_DATA["worker_replicas"]
     compute_machine_type = config.ENV_DATA["worker_instance_type"]
+    multi_az = "--multi-az " if config.ENV_DATA["multi_availability_zones"] else ""
     cmd = (
         f"rosa create cluster --cluster-name {cluster_name} --region {region} "
         f"--compute-nodes {compute_nodes} --mode auto --compute-machine-type "
-        f"{compute_machine_type}  --version {rosa_ocp_version} --sts --yes"
+        f"{compute_machine_type}  --version {rosa_ocp_version} {multi_az}--sts --yes"
```

Comment:
I would propose to add space between `{multi_az}` and `--sts` parameters. I know, that the space is added after the `--multi_az` parameter directly, but I think it will be little bit more readable, to have the space here (like it will not look like the `{multi_az}--sts` belongs together).
```suggestion
        f"{compute_machine_type}  --version {rosa_ocp_version} {multi_az} --sts --yes"
```

Reply:
That will add extra space in the command string in case of multi_availability_zones is False.